core/build.gradle

```diff
@@ -182,4 +187,15 @@ integTest {
     // testLogging.showStandardStreams = true
 }
 
+bench {
+    systemProperty "spark.ext.h2o.backend.cluster.mode", detectBackendClusterMode()
+
+    systemProperty "spark.testing",   "true"
+    systemProperty "spark.test.home", "${sparkHome}"
+
+    // Pass list of jars required for testing
+    systemProperty "sparkling.assembly.jar", "${project(":sparkling-water-assembly").configurations.shadow.artifacts.file.join(',')}"
+    systemProperty "sparkling.bench.jar", "${benchJar.archivePath}"
```

Comment:
The property `sparkling.bench.jar` is not used anywhere in the code, I think it can be removed.

Also if I'm correct, current bench test don't require `sparkling.assembly.jar` to be set